doc/visual-programming/source/widgets/data/pivot.md

```diff
@@ -0,0 +1,68 @@
+3. Values that will be used for aggregation. Aggregated values will appear as cells in the table.
+4. Aggregation methods:
+   - For any variable type:
+      - *Count*: number of instances that appear in the data
+      - *Count defined*: number of non-empty (not NaN) instances in the data.
+   - For numeric variables:
+      - *Sum*: sum of values.
+      - *Mean*: average of values.
+      - *Mode*: most frequent value of the subset.
+      - *Min*: smallest value.
+      - *Max*: highest value.
+      - *Median*: middle value.
+      - *Var*: variance of the subset.
+   - For discrete variables:
+      - *Majority*: most frequent value of the subset.
+5. Tick the box on the left to automatically output any changes. Alternatively, press *Apply* .
+
+Discrete variables
+------------------
+
+![](images/Pivot-discrete.png)
+
+Example of a pivot table with only discrete variables selected. We are using *heart-disease* data set for this example. We are using the values of *diameter narrowing* as row values, namely 0 and 1. Our columns are values of *gender*, namely female and male. We are using *thal* as values in our cells.
+
+We have selected *Count* and *Majority* as aggregation methods. In the pivot table, we can see the number of instances that do not have diameter narrowing and are female. There are 72 such patients. Concurrently, there are 92 male patients that don't have diameter narrowing. Thal values don't have any effect here, we are just counting occurrences in the data.
+
+The second row shows majority. This means most female patients that don't have diameter narrowing have normal thal results. Conversely, female patients that have diameter narrowing most often have reversable defect.
+
+Numeric variables
+-----------------
+
+![](images/Pivot-continuous.png)
+
+Example of a pivot table with numeric variables. We are using *heart-disease* data set for this example. We are using the values of *diameter narrowing* as row values, namely 0 and 1. Our columns are values of *gender*, namely female and male. We are using *rest SBP* as values in our cells.
+
+We have selected *Count*, *Sum* and *Median* as aggregation methods. Under *Count*, we see there are 72 female patients that don't have diameter narrowing, same as before for discrete values. What is different are the sum and median aggregations. We see that the sum of resting systolic blood pressure for female patients that don't have diameter narrowing is 9269 and the median value is 130.
+
+Example
+-------
+
+We are using *Forest Fires* for this example. The data is loaded in the [Datasets](../data/datasets.md) widget and passed to **Pivot Table**. *Forest Fires* datasets reports forest fires by the month and day they happened. We can aggregate all occurrences of forest fires by selecting *Count* as aggregation method and using *month* as row and *day* as column values. Since we are using *Count*, it does not matter what our *Values* variable will be, so we will leave it as is.
```

Comment:
`it does not matter what our *Values* variable will be` -> maybe `*Values* is unimportant` (or something similar)